### PR TITLE
Feature/destroybuttons

### PR DESCRIPTION
**Pull request by mplattu**
Adds buttons for
 * removing exams (`vagrant destroy -f`)
 * removing server (deleting Vagrant and VirtualBox directories)

### CODE REVIEW

File: src/naksu/mebroutines/remove/remove.go [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+import (
+  "fmt"
+  "errors"
+	"naksu/mebroutines"
+	"naksu/progress"
+)
+
+// Server removes all directories related to Vagrant and VirtualBox
+func Server() error {
```
Comment (hsalokor) [inline src/naksu/mebroutines/remove/remove.go]: I'd name this `RemoveServer()` since it would make the context clearer

File: src/naksu/ui.go [hunk 2]
```diff
@@ -47,12 +56,35 @@ var backupLabel *ui.Label
...
+var removeInfoLabel [5]*ui.Label
+
+
 func createMainWindowElements() {
 	// Define main window
-	buttonStartServer = ui.NewButton(xlate.Get("Start Stickless Exam Server"))
-	buttonGetServer = ui.NewButton("Install or update Abitti Stickless Exam Server")
-	buttonSwitchServer = ui.NewButton("Install or update Stickless Matriculation Exam Server")
-	buttonMakeBackup = ui.NewButton("Make Stickless Exam Server Backup")
+	buttonStartServer = ui.NewButton(xlate.Get("Start Exam Server"))
```
Comment (hsalokor) [inline src/naksu/ui.go]: This seems out of place? If I recall correctly the translations are updated later on so `xlate.Get(...)` is not necessary here

File: src/naksu/mebroutines/remove/remove.go [hunk 3]
```diff
@@ -0,0 +1,59 @@
+package remove
+
+import (
+  "fmt"
+  "errors"
```
Comment (hsalokor) [inline src/naksu/mebroutines/remove/remove.go]: Indent is off here

File: src/naksu/mebroutines/mebroutines.go [hunk 4]
```diff
@@ -308,6 +262,12 @@ func CreateFile(path string) error {
 	return err
 }
 
+// RemoveDir removes directory and all its contents
+func RemoveDir(path string) error {
```
[the remark this answers is not in the document]
Comment (mplattu) [reply src/naksu/mebroutines/mebroutines.go]: I acree, but It's aligned with existing `mebroutines.CreateDir`, `mebroutines.CreateFile` etc.

File: src/naksu/mebroutines/remove/remove.go (hunk 1 above)
Comment (mplattu) [reply src/naksu/mebroutines/remove/remove.go]: lint does not like your suggestion: `func name will be used as remove.RemoveServer by other packages, and that stutters; consider calling this Server`

**Pull request merged**